src/gsi/prewgt_reg.f90

```diff
@@ -711,6 +758,11 @@ subroutine prewgt_reg(mype)
 
 ! Set up scales
 
+  if( if_cs_staticB )then
+     bin_l = 2 ! temporily assign the scale in second bin to all bins 
+  else
+     bin_l = 1
+  end if
```

Comment:
> This bin_l is not used. I think it can be deleted.

This part includes bugs when we merged the codes. Please see the latest codes.

Reply:
I saw the latest code, but why is bin_l=2 selected in if_cs_staticB?

Reply:
This is a temporary option for the recursive filter, where the same length scale is used in the entire domain. Because we use convective-scale static B in the storm-scale assimilation only, it is still safe to use the weak-storm length scale for the entire domain.

Reply:
If so, bin_l=1 also should be safe. Why is it required to change it from 1 to 2?